vatools/vcf_readcount_annotator.py

```diff
@@ -95,6 +95,15 @@ def is_insertion(ref, alt):
 def is_deletion(ref, alt):
     return len(alt) < len(ref)
 
+def has_complex_variant(entry):
+    if has_indel(entry):
+        for alt in entry.ALT:
+            alt = alt.serialize()
+            ref = entry.REF
+            if len(ref) == len(alt) or (len(ref) > 1 and len(alt) > 1):
```

Comment:
this won't work for complex indels of the type `AAAA   TTTT`.  has_indel will never trip, because it requires len(ref) != len(alt) on line 88

Reply:
Good catch. I adjusted the `has_indel` and `has_snv` subroutines and added a test case for MNPs